src/components/styled-atlaskit/DynamicTable/DynamicTable.tsx

```diff
@@ -14,12 +14,13 @@ export interface DynamicTableProps extends PropsWithChildren<StatelessProps> {
   className?: string;
   customContent?: React.ReactNode;
   dataQa?: string | null;
+  noPadding?: boolean;
 }
 
 export const DynamicTable = (props: DynamicTableProps) => {
-  const { className, children, customContent = null, dataQa = null } = props;
+  const { className, children, customContent = null, dataQa = null, noPadding = false } = props;
   return (
-    <div className={classNames("styled-atlaskit-table", className)} data-qa-id={dataQa}>
+    <div className={classNames("styled-atlaskit-table", { "no-padding": noPadding }, className)} data-qa-id={dataQa}>
```

Comment:
does `no-padding` class already being handled by css?

Reply:
what do you mean by handled by css? i saw that some dynamic table in AP has no-padding in its classname, so i thought it's better to add noPadding prop to conditionally add no-padding beside styled-atlaskit-table

Reply:
okay then, just found the no-padding class attach to dynamic table